src/deye_sensors_igen_dtsd422.py

```diff
@@ -54,15 +54,16 @@
     0x0F,
     1,
     mqtt_topic_suffix="ct1/active_power",
-    unit="W",
+    unit="W", device_class='power',
     groups=["igen_dtsd422"],
 )
 ct1_reactive_power_sensor = SignedMagnitudeDoubleRegisterSensor(
     "Reactive Power CT1",
     0x17,
     1,
     mqtt_topic_suffix="ct1/reactive_power",
-    unit="Var",
+    unit="var",
```

Comment:
Is this change required by HA?

Reply:
Yes, and it is the si unit for reactive power